src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -188,9 +188,14 @@ public static SvnInfoResult GetSvnFileInfo(this ICakeContext context, FilePath f
         /// <param name="context">The Cake context.</param>
         /// <param name="filePath">The file.</param>
         /// <param name="settings">The settings.</param>
-        /// <returns>The Subversion information.</returns>
+        /// <returns>The Subversion information. If no information about the file is available on Subversion, 
+        /// an empty SvnInfoResult will be returned. For this <see cref="SvnSettings.ThrowOnError"/> has to be set to <c>false</c> 
```

Comment:
Doesn't this require https://github.com/cake-contrib/Cake.Svn/issues/42 to be set? `SvnSettings.ThrowOnError` isn't implemented anywhere yet.

Reply:
yes, #42 is also required